cob_teleop/common/include/cob_teleop/cob_teleop_v2_common.h

```diff
@@ -2,14 +2,6 @@
 #include "ros/ros.h"
 #include <sensor_msgs/JoyFeedbackArray.h>
 #include <geometry_msgs/Twist.h>
-#include <geometry_msgs/Twist.h>
-#include <geometry_msgs/Twist.h>
-#include <std_msgs/Float64MultiArray.h>
-#include <std_msgs/Float64MultiArray.h>
-#include <geometry_msgs/Twist.h>
-#include <std_msgs/Float64MultiArray.h>
-#include <geometry_msgs/Twist.h>
-#include <std_msgs/Float64MultiArray.h>
```

Comment:
Can you do the same cleanup [here](https://github.com/ipa320/cob_command_tools/blob/indigo_dev/cob_teleop/ros/src/cob_teleop_v2_ros.cpp#L7-L18)?